tests/unit/amo/components/TestAddonQRCodeLink.js

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+
+import AddonQRCode from 'amo/components/AddonQRCode';
+import AddonQRCodeLink, {
+  AddonQRCodeLinkBase,
+} from 'amo/components/AddonQRCodeLink';
+import { CLIENT_APP_ANDROID, CLIENT_APP_FIREFOX } from 'amo/constants';
+import {
+  createFakeEvent,
+  createInternalAddonWithLang,
+  dispatchClientMetadata,
+  fakeAddon,
+  fakeI18n,
+  getFakeConfig,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  const goodAddonId = 12345;
+  const testConfig = { addonIdsWithQRCodes: [goodAddonId] };
+  const goodAddon = createInternalAddonWithLang({
+    ...fakeAddon,
+    id: goodAddonId,
+  });
```

Comment:
This code should probably be:

```suggestion
  const addonIdsWithQRCodes = [12345];
  const addonWithQRCode = createInternalAddonWithLang({
    ...fakeAddon,
    id: addonIdsWithQRCodes[0],
  });
```

Reply:
I agree that naming is clearer. I will make that change.